example/main.tf

```diff
@@ -56,8 +56,8 @@ data "aws_subnets" "private" {
 
     ## try the created subnets from the upstream network module, or override with custom names
     values = length(var.subnet_names) > 0 ? var.subnet_names : [
-      "${var.namespace}-${var.environment}-private-${var.region}a",
-      "${var.namespace}-${var.environment}-private-${var.region}b"
+      "${var.namespace}-${var.environment}-private-subnet-private${var.region}a",
+      "${var.namespace}-${var.environment}-private-subnet-private${var.region}b"
```

Comment:
@mayank0202 please confirm the naming convention in arc/poc account, but I believe it is now: `${var.namespace}-${var.environment}-private-subnet-private-${var.region}a",`

Reply:
@jamescrowley321  @tsaucier-sf  as i mentioned that i tested this on poc account so this naming convention is in poc account thats why i needed to changed this

Reply:
We need to take a look then because that account should have the updated naming conventions, which were there previously.

Reply:
You are missing a hyphen here: `private${var.region}b`
It worked in poc because it's named correctly in `variables.tf`. 
Please fix this typo.